.gitlab-ci.yml

```diff
@@ -7,19 +7,20 @@ stages:
 - test
 - publish
 
+.common_refs: &common_refs
```

Comment:
Don't really like the name, but I can't think of any better ones.
`acceptable_brances`? `only_refs` (that one's kind of bad too)?

The [documentation](https://docs.gitlab.com/ce/ci/yaml/#onlyexcept-basic) doesn't suggest anything useful. Though it _does_ say that it'll be deprecated soon…

Reply:
I'll think in a better label for this anchor and move to rules in a subsequent task.